### DISCUSSION TITLE
Empty Message.content

### DISCUSSION THREAD

**Discussion by iHazzu** [Rapptz/discord.py#6832] [Q&A]
I have a user bot and recently it stopped responding to commands because it was receiving null message content. The Message object still has a channel and an author normally, but the content is always empty.
Here is the bot object:
```py
...
bot = commands.Bot(
  command_prefix='$',
  intents=discord.Intents(guilds=True, messages=True, reactions=True),
  case_insensitive=True
)
...
bot.run(env['TOKEN'], bot=False)
```
I'm using the version 1.7.1 of discord.py

**Comment by Rapptz** [comment]
User-bots and self-bots have been deprecated in the library since v1.7 and will no longer receive support. As of v2.0, they have been completely removed from the library as well.
